modules/Sample/BodyPart.yaml

```diff
@@ -137,6 +137,9 @@ enums:
       pancreas:
         description: A large lobulated gland of vertebrates that secretes digestive enzymes and the hormones insulin and glucagon.
         meaning: BTO:0000988
+      placenta:
+        description: The vascular organ in mammals except monotremes and marsupials that unites the fetus to the maternal uterus and mediates its metabolic exchanges through a more or less intimate association of uterine mucosal with chorionic and usually allantoic tissues; also: an analogous organ in another animal.
```

Comment:
If you use ":" as in "...allantoic tissues; also: an analogous organ in another animal...", you must quote the description.